testing/mame/APKBUILD

```diff
@@ -0,0 +1,165 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=mame
+pkgver=0.200
+pkgrel=0
+pkgdesc="Multi Arcade Machine Emulator with GroovyMAME/Switchres/No-nag patchset."
+url="http://mamedev.org"
+arch="all"
+license="GPL-2.0-or-later"
+depends="$pkgname-common"
+makedepends="coreutils dos2unix nasm clang-dev libxrandr-dev libxinerama-dev
+	alsa-lib-dev python2-dev py3-sphinx sdl2-dev sdl2_ttf-dev expat-dev zlib-dev
+	mesa-dev libxinerama-dev fontconfig-dev libjpeg-turbo-dev flac-dev lua5.3-dev
+	sqlite-dev portaudio-dev rapidjson-dev glm-dev portmidi-dev asio-dev"
+subpackages="$pkgname-data::noarch $pkgname-doc $pkgname-common::noarch $pkgname-lang
+	$pkgname-tools $pkgname-plugins::noarch $pkgname-arcade $pkgname-mess"
+_groovymame_patch=groovymame-${pkgver}.diff
+source="https://github.com/mamedev/${pkgname}/archive/${pkgname}${pkgver/.}.tar.gz
```

Comment:
please change source to `https://github.com/mamedev/mame/archive....` to easily get url of releases